app/sign-up.tsx

```diff
@@ -24,9 +25,15 @@ export default function SignUpPage() {
       return;
     }
 
-    await signUp(data.email, data.password);
-    if (!error) {
-      router.replace('/(auth)');
+    try {
+      const message = await signUp(data.email, data.password);
+      // Handle the message if it's defined
+      if (message) {
+        setSignUpMessage(message);
+      }
+    } catch (error) {
+      // Handle any sign-up errors here
+      console.error('Sign-up error:', error);
```

Comment:
I think it would be a good idea to show this text so the user can see if there's an error (in addition to logging to the console)

Reply:
Error handling is already done in the useAuthStore's signUp function, which sets the error state, and that is displayed to the screen 

`              {error && <Text className="text-red-500 mt-2 text-center">{error}</Text>}
`

I'll remove this console.log and try-catch block, if we need the logs for errors I think it might be best to handle them in useAuthStore?